package.json

```diff
@@ -56,7 +56,7 @@
     "@types/compression": "^1.7.1",
     "@types/node": "18",
     "compression": "^1.7.4",
-    "jsdom": "^22.1.0",
```

Comment:
jsdom has a bug in `URLSearchParams.prototype.delete`. It ignores explicit `value` argument when dealing with multiple key/value pairs.